trail/upload/process_uploads/header_standardizer.py

```diff
@@ -320,6 +379,50 @@ def standardizeWcs(self, hdu=None):
             header = hdu.header
         else:
             header = self.header
+        return header, dimX, dimY
 
-        wcs = models.Wcs(**self._computeStandardizedWcs(header, dimX, dimY))
-        return wcs
+    @staticmethod
+    def _astrometryNetSolver(path_to_file):
+        """Given a fits file it will process and send to astrometry.net
+        where it will obtain the WCS data for the file if it is able to.
+        If not then it will return an empty dictionary
+        (this can be altered to whatever needs to be used for)
```

Comment:
Is this correct? It looks like it will either raise or return 3 things (header dict, dimX, dimY). Please update the Returns block and description to be self consistent.